src/modules/Schedule/components/ScheduleTable/index.tsx

```diff
@@ -1,51 +1,43 @@
-import styled from '@emotion/styled'
-import { useCallback } from 'react'
-import { DragDropContext, Droppable, DropResult } from 'react-beautiful-dnd'
+import { lazy, Suspense, useEffect, useState } from 'react'
 
 import { useCourseGroup } from '@/common/hooks/useCourseGroup'
 import { CourseCart } from '@/store'
 
 import { CourseOverlapMap } from '../Schedule/utils'
 import { ScheduleTableCard } from './components/ScheduleTableCard'
+import { CardLayout } from './components/ScheduleTableCard/styled'
+import { Layout } from './styled'
 
 export interface ScheduleTableProps {
   courseCart: CourseCart
   overlappingCourses: CourseOverlapMap
 }
 
-const Layout = styled.div`
-  padding-bottom: 100px;
-`
+const InteractiveScheduleTable = lazy(() => import('./InteractiveScheduleTable'))
```

Comment:
why dont you use `dynamic` na?

Reply:
How about https://nextjs.org/docs/advanced-features/dynamic-import#with-suspense ?

Reply:
The suspense prop doesn't exist in our version

Reply:
It's probably added in next 12 , which you haven't merge yet